src/components/ActiveClients.js

```diff
@@ -133,7 +133,7 @@ const ActiveClients = () => {
               return (
                 <tr key={key}>
                   <td>
-                    <Link className='text-decoration-none' to={`/pings/${debugId}`}>
+                    <Link className='text-decoration-none' to={`/pings/${debugId}`} onClick={recordClick('Debug ID')}>
```

Comment:
I think this is causing the issue I mentioned in the previous comment. This function is being immediately invoked on render, which is causing every single row in `ActiveClients` to call `recordClick()` on mount.

This needs to be `onClick={() => recordClick('Debug ID')}`. In the rest of the places that we are using `recordClick` we are doing so inside of a callback function, since this isn't inside of a callback you either need to make this a function that calls a function (not necessary here) or just wrap it in a callback.

Reply:
Thanks for catching this bug. Yeah, it was a miss. I fixed it in the next commit.